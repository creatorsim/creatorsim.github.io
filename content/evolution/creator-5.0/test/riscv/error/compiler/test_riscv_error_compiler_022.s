
#
# Creator (https://creatorsim.github.io/creator/)
#

 .data
    .zero -10
    
