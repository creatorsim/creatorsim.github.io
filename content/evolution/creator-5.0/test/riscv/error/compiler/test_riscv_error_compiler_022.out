

[E21] Error: Negative values aren't allowed here
   ╭─[ assembly:7:11 ]
   │
 7 │     .zero -10
   │           ─┬─  
   │            ╰─── This expression has value `-10`
───╯

Not executed
keyboard[0x0]:''; display[0x0]:''; 

